### DISCUSSION TITLE
cannot build due no account for team "43AQ936H96" and signing issue.

### DISCUSSION THREAD

**Discussion by IndpendentLLC** [mozilla-mobile/firefox-ios#8788] [Q&A] [answered]
complete the steps in the readme page, and then run the bootstrap.sh through terminal and then copy the git to the computer. No matter whichever version I try launching on my iPhone, it cannot build due to these errors. 
No account for team "43AQ936H96". Add a new account in the Accounts preference pane or verify that your accounts have valid credentials. 

No profiles for 'org.mozilla.ios.Fennec.ShareTo' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'org.mozilla.ios.Fennec.ShareTo'.

Is there a profile from Mozilla I need to download and/or login info I need? 

latest public build of Xcode and MacOS 11.4

Thank you for help. 
Dre~

**Comment by rogeriochaves** [comment, marked as answer]
hey there! I was having the same problem, I tried to fix it with using my own team manually on Xcode UI, withouth much success, but then, I found something that worked:

I opened the project in Sublime Text, did a search and replace all over for 43AQ936H96 (their team code) to my own team code, also I replaced all "org.mozilla.ios" with "app.rchaves.ios"

This finally allowed the app to compile!

**Reply by njpig** [reply]
Thanks a lot!

After following what you did, I did a successful build. BTW, I replaced all "org.mozilla.ios" with "my.firefoxbuid.ios"(which has to be unique.
